tests/fixtures/extended-event-input.json

```diff
@@ -161,5 +161,5 @@
         "username": "username"
     },
     "version": "7",
-    "location": null
+    "location": "not-empty-so-that-it-isnt-dropped"
```

Comment:
Extra fields with null values are not serialized. Setting this value to anything else ensures we test future-compatibility.